### ISSUE TITLE
Extend HinSAGE for Link Prediction

### ISSUE THREAD

**Issue by kjun9** [stellargraph/stellargraph#30]
### Description
Use the documentation for HinSAGE link prediction to create a working link prediction example using the Paradise Papers dataset from the Data team.

### User Story
As a: data scientist
I want: to use GraphSAGE layers for link prediction
so that: I can run scalable link prediction

### Done Checklist (Development)
- [ ] Working example with Alzheimer data
- [ ] Code well commented
- [ ] Documentation in repo
- [ ] Peer Code Review Performed

**Comment by youph** [comment]
@kjun9 is this still relevant?

**Comment by kjun9** [comment]
I guess if the library intends to continue only providing the "base" HinSAGE layers and allow the user to stack whatever loss function they would like with Keras' framework, then this issue doesn't need to exist.

**Comment by kjun9** [comment]
Maybe the issue makes more sense as something like "Create a link prediction example with HinSAGE" if you don't have that already
